src/openforms/analytics_tools/contrib/expoints/csp_headers.json

```diff
@@ -13,7 +13,7 @@
     },
     {
         "directive" : "font-src",
-        "value": "https://cdn.expoints.nl"
+        "value": "'self' https://cdn.expoints.nl"
```

Comment:
This doesn't seem to work for me:

```
Content-Security-Policy: The page’s settings blocked the loading of a resource (font-src) at http://localhost:8000/static/sdk/static/media/fa-brands-400.e033a13ee751afc1860c.woff2 because it violates the following directive: “font-src https://*.expoints.nl https://cdn.expoints.nl/”
```

with these directives

![image](https://github.com/user-attachments/assets/70164aad-3d1f-4fa8-ab39-335254db7a60)